installation/proc-install-rhdh-operator.adoc

```diff
@@ -0,0 +1,34 @@
+[id='proc-install-rhdh-operator_{context}']
+= Installing {product} using the Operator
+
+[IMPORTANT]
+====
+The {product} Operator is a Technology Preview feature only.
+
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
+====
+
+This section describes the installation procedure for installing {product} on your OpenShift Container Platform using the Operator.
+
+//You can use the {product} Operator in Red Hat OpenShift (OpenShift) to install {product}.
+
+.Prerequisites
+
+* You are logged in as an administrator on the OpenShift web console.
+* You have configured the appropriate roles and permissions within your project to create an application. See the link:https://docs.openshift.com/container-platform/4.14/applications/index.html[Red Hat OpenShift documentation on Building applications] for more details.
+* If installing a CI build from quay.io/rhdh, you have added a link:https://gist.github.com/nickboldt/d54ba9d7264a5480d14eea6c3bd2ecdf[RHDH Operator Catalog Source] and a link:https://gist.github.com/nickboldt/84bea28da11ec5aab549c182818356e0#file-oc-pull-secret-update-project-sh[pull secret].
+
+.Procedure
+. In the *Administrator* view of the OpenShift web console, go to *Operators* → *OperatorHub*, search for Red Hat Developer Hub, and install the Red Hat Developer Hub Operator. For more info, see link:https://docs.openshift.com/container-platform/4.14/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-from-operatorhub-using-web-console_olm-adding-operators-to-a-cluster[Installing from OperatorHub using the web console].
```

Comment:
Administrator view => Administrator perspective
Also instead of using a reference link for operator installation, it might be more helpful  for users if we include the operator installation steps here directly with screenshots.